lxqt-base/lxqt-meta/lxqt-meta-0.15.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PV="$(ver_cut 1-2)*"
+
+DESCRIPTION="Meta ebuild for LXQt, the Lightweight Desktop Environment"
+HOMEPAGE="https://lxqt.org/"
+
+if [[ ${PV} != *9999* ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+fi
+
+LICENSE="metapackage"
+SLOT="0"
+
+IUSE="+about admin archiver +filemanager lightdm lximage minimal
+	nls	+policykit powermanagement processviewer screenshot sddm
```

Comment:
I know you have been thinking about lxqt-meta use flags lately, this is just a suggestion taken from plasma-meta: `+display-manager` if you think having a display manager by default would make sense; and then only `sddm` to switch between that and lightdm. Preference on whatever upstream may recommend or you think fits best.

Reply:
Yeah, seems like it would make sense.  Will likely try it out.